data-service/src/main/java/com/amazonaws/blox/dataservice/repository/model/EnvironmentRevisionDDBRecord.java

```diff
@@ -52,8 +52,9 @@ public static EnvironmentRevisionDDBRecord withKeys(
 
   @DynamoDBVersionAttribute private Long recordVersion;
 
-  //encoded name:value
+  //encoded name_value
   private Set<String> attributes;
```

Comment:
Any reason not to use the DynamoDBTypeConverter and declare this as a `Set<Attribute>`?

Reply:
sure